HerPortal/Startup.cs

```diff
@@ -75,19 +75,26 @@ public void ConfigureServices(IServiceCollection services)
             })
             .AddCookie(options =>
             {
+                options.ExpireTimeSpan = TimeSpan.FromMinutes(15);
+                options.SlidingExpiration = true;
                 options.Cookie.HttpOnly = true;
                 options.Cookie.SecurePolicy = CookieSecurePolicy.Always;
+                options.Cookie.SameSite = SameSiteMode.Lax;
             })
             .AddOpenIdConnect(options =>
             {
-                options.ResponseType = "code";
+                options.NonceCookie.SameSite = SameSiteMode.Lax;
+                options.CorrelationCookie.SameSite = SameSiteMode.Lax;
```

Comment:
Do we think setting these to `Lax` is doing anything (if the default is `None`?)

Reply:
It's making the cookies slightly more secure, but I don't have a concrete scenario that it helps with